test-files/terraform/azure/compute/az-nsg.tf

```diff
@@ -0,0 +1,26 @@
+provider "azurerm" {
```

Comment:
Do we use these Azure or GCP files for anything? If not, can you please remove them?